src/main/java/pl/project13/jgit/JGitCommon.java

```diff
@@ -97,16 +80,16 @@
     }
   }
 
-  public String getClosestTagName(@NotNull LoggerBridge loggerBridge,@NotNull Repository repo){
-    Map<ObjectId, List<DatedRevTag>> map = getClosestTagAsMap(loggerBridge,repo);
+  public String getClosestTagName(@NotNull Repository repo, @NotNull Mojo mojo){
```

Comment:
I was thinking about this, but the advantages are not very clear to me. Anyway, then this needs another round of refactoring to reintroduce the bridge.